python/audit/management/commands/clearaudit.py

```diff
@@ -26,107 +26,107 @@
-            mode='w:gz',
-            encoding='utf-8',
+            mode="w:gz",
+            encoding="utf-8",
             compresslevel=9,
         ),
     )
 
     archive_model_postfix_map = {
-        AuditLog: 'operations',
-        AuditSession: 'logins',
-        AuditObject: 'objects',
+        AuditLog: "operations",
+        AuditSession: "logins",
+        AuditObject: "objects",
     }
 
     def handle(self, *args, **options):
         try:
             _, config = get_adcm_config(self.config_key)
-            if config['retention_period'] <= 0:
-                self.__log('Disabled')
+            if config["retention_period"] <= 0:
+                self.__log("Disabled")
                 return
 
-            threshold_date = timezone.now() - timedelta(days=config['retention_period'])
-            self.__log(f'Started. Threshold date: {threshold_date}')
+            threshold_date = timezone.now() - timedelta(days=config["retention_period"])
+            self.__log(f"Started. Threshold date: {threshold_date}")
 
             # get delete candidates
             target_operations = AuditLog.objects.filter(operation_time__lt=threshold_date)
             target_logins = AuditSession.objects.filter(login_time__lt=threshold_date)
             objects_pk_to_delete = set()
             for ao in AuditObject.objects.filter(is_deleted=True):
                 if not ao.auditlog_set.exclude(
-                    pk__in=target_operations.values_list('pk', flat=True)
+                    pk__in=target_operations.values_list("pk", flat=True)
                 ).exists():
                     objects_pk_to_delete.add(ao.pk)
             target_objects = AuditObject.objects.filter(pk__in=objects_pk_to_delete)
 
             cleared = False
             if any(qs.exists() for qs in (target_operations, target_logins, target_objects)):
-                make_audit_log('audit', 'success', 'launched')
+                make_audit_log("audit", "success", "launched")
 
-            if config['data_archiving']:
+            if config["data_archiving"]:
                 archive_path = os.path.join(self.archive_base_dir, self.archive_name)
-                self.__log(f'Target audit records will be archived to `{archive_path}`')
+                self.__log(f"Target audit records will be archived to `{archive_path}`")
                 self.__archive(target_operations, target_logins, target_objects)
             else:
-                self.__log('Archiving is disabled')
+                self.__log("Archiving is disabled")
 
             cleared = self.__delete(target_operations, target_logins, target_objects)
 
-            self.__log('Finished.')
+            self.__log("Finished.")
             if cleared:
-                make_audit_log('audit', 'success', 'completed')
+                make_audit_log("audit", "success", "completed")
 
         except Exception as e:  # pylint: disable=broad-except
```

Comment:
It's better to minimize lines under `try` and use specific exeptions

Reply:
i think broad exception is needed to make correct auditlog

Reply:
partially fixed